specification/dartLangSpec.tex

```diff
@@ -16245,18 +16246,20 @@ \subsection{Assignable Expressions}
 \LMLabel{assignableExpressions}
 
 \LMHash{}%
-Assignable expressions are expressions
+\IndexCustom{Assignable expressions}{assignable expression} are terms
 that can appear on the left hand side of an assignment.
-This section describes how to evaluate these expressions
+This section describes how to evaluate these terms
 when they do not constitute the complete left hand side of an assignment.
```

Comment:
Maybe add: "\commentary{That is, when they are being evaluated for their value, not being assigned to.}"
Or just add "to a value" to the "evaluated" below. Just for clarity.